### PR TITLE
Cleanup tasks and js dependencies

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR cleanup and simplify JS tasks and depdencies:
- added a `.nvmrc` (same version as udata)
- dropped all useless depdencies
- ensure invoke tasks are using webpack from node_modules
- upgrade js versions

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -38,54 +38,52 @@ def clean(bower=False, node=False):
     ]
     for pattern in patterns:
         print('Removing {0}'.format(pattern))
-        run('cd {0} && rm -rf {1}'.format(ROOT, pattern))
+        lrun('rm -rf {0}'.format(ROOT, pattern))
```
Comment (davidbgk) [inline tasks.py]: Isn't it dangerous to launch a `rm -rf` at an unidentified place? + `format()` contains now an extra parameter.

**Comment by noirbizarre** [comment]
Is it good to merge ?

File: tasks.py (hunk 1 above)
Comment (noirbizarre) [inline tasks.py]: it was a typo, fixed

**Comment by davidbgk** [comment]
LGTM!

**Pull request merged**